test/conformance/conformance_flowgraph.h

```diff
@@ -17,74 +17,810 @@
+    mutable std::size_t assignes_count;
+    std::size_t move_count;
+    bool is_copy;
+
+    counting_object():
+        copy_count(0), copies_count(0), assign_count(0),
+        assignes_count(0), move_count(0), is_copy(false) {}
+
+    counting_object(int):
+        copy_count(0), copies_count(0), assign_count(0),
+        assignes_count(0), move_count(0), is_copy(false) {}
+
+    counting_object( const counting_object<OutputType, InputType>& other ):
+        copy_count(other.copy_count + 1), is_copy(true) {
+            ++other.copies_count;
+        }
+
+    counting_object& operator=( const counting_object<OutputType, InputType>& other ) {
+        assign_count = other.assign_count + 1;
+        ++other.assignes_count;
+        is_copy = true;
+        return *this;
+    }
+
+    counting_object( counting_object<OutputType, InputType>&& other ):
+         copy_count(other.copy_count), copies_count(other.copies_count),
+         assign_count(other.assign_count), assignes_count(other.assignes_count),
+         move_count(other.move_count + 1), is_copy(other.is_copy) {}
+
+    counting_object& operator=( counting_object<OutputType, InputType>&& other ) {
+        copy_count = other.copy_count;
+        copies_count = other.copies_count;
+        assign_count = other.assign_count;
+        assignes_count = other.assignes_count;
+        move_count = other.move_count + 1;
+        is_copy = other.is_copy;
+        return *this;
+    }
+
+    OutputType operator()( InputType ) {
+        return OutputType(1);
+    }
+
+    void operator()( InputType, multifunc_ports_t<InputType,OutputType>& op ) {
+        std::get<0>(op).try_put(OutputType(1));
+    }
+
+    void operator()( InputType , assync_ports_t<InputType, OutputType>& g) {
+        g.try_put(OutputType(1));
+    }
+
+    OutputType operator()( oneapi::tbb::flow_control & fc ) {
+        static bool check = false;
+        if(check) {
+            check = false;
+            fc.stop();
+            return OutputType(1);
+        }
+        check = true;
+        return OutputType(1);
+    }
+};
+
+template <typename OutputType = int>
+struct passthru_body {
+    OutputType operator()( const oneapi::tbb::flow::continue_msg& ) {
+        return OutputType(0);
+    }
+
+    OutputType operator()( const OutputType& i ) {
+        return i;
+    }
+
+    OutputType operator()( oneapi::tbb::flow_control & fc ) {
+        static bool check = false;
+        if(check) {
+            check = false;
+            fc.stop();
+            return OutputType(0);
+        }
+        check = true;
+        return OutputType(0);
+    }
+
+    void operator()( OutputType argument, multifunc_ports_t<OutputType>& op ) {
+        std::get<0>(op).try_put(argument);
+    }
+
+    void operator()( OutputType argument, assync_ports_t<OutputType>& g ) {
+        g.try_put(argument);
+    }
+};
+
+template<typename Node, typename InputType = void>
+bool produce_messages(Node& node, int arg = 1) {
```

Comment:
`produce_messages` is too far from the corresponding `get_values` function. I think they need be placed nearby in the file.